packages/create-next-app/templates/api/js/api/index.js

```diff
@@ -0,0 +1,3 @@
+export default (req, res) => {
+    return res.json({ message: "Hello World" });
+};
```

Comment:
cc @ephraimduncan

This is not right. We are aiming to use the App Router's [route.js](https://nextjs.org/docs/app/api-reference/file-conventions/route).